src/geofabrics/processor.py

```diff
@@ -219,9 +219,16 @@ def get_lidar_dataset_crs(self, data_service, dataset_name) -> dict:
             return None
 
     def get_lidar_file_list(self, data_service) -> dict:
-        """ Return a dictionary that contains a list of LiDAR tiles to process under the key 'file_paths' and optionally
-        the CRS of these files under the key 'crs'. The 'crs' should only be set if the CRS information is not correctly
-        encoded in the LAZ/LAS files.
+        """ Return a dictionary with three enties 'file_paths', 'crs' and 'tile_index_file'. The 'file_paths' contains a
+        list of LiDAR tiles to process.
+
+        The 'crs' is only optionally set (if unset the value is None). The 'crs' should only be set if the CRS
```

Comment:
What crs mean?

Reply:
I'll copy in the longer comment from elsewhere. Sorry I should have included it in this docstring. CRS effectively means the coordinate system

Reply:
I have changed to "The 'crs' (or coordinate system of the LiDAR data as defined by an EPSG code) is only optionally set (if unset the value is None). The 'crs' should only be set if the CRS information is not correctly encoded in the LAZ/LAS  files. Currently this is only supported for OpenTopography LiDAR." and will push this commit along with any other edits you pick-up.